src/harness/reference_models/iap/iap.py

```diff
@@ -198,75 +200,77 @@ def iapPointConstraint(protection_point, channels, low_freq, high_freq,
   grants_overlap_flag = {}
   grants_interference = {}
 
-  while num_unsatisfied_grants > 0:
-    for g_idx, grant in enumerate(neighbor_grants):
+  with cache.CacheManager(wf_hybrid.CalcHybridPropagationLoss):
```

Comment:
even though diff showed significant differences, I guess this is the only addition added, right ? Same comment on the other file.

Reply:
Check https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/pull/732/files?w=1

Reply:
Good, thanks.